docs/en/plan-deployment/cost.md

```diff
@@ -30,7 +30,7 @@ The following are cost estimations for monthly data volumes of 10/100/1000 RPS (
 
 Ingestion module includes the following cost components:
 
-- Application load balancer
+- Application load balancer and Public IP
```

Comment:
-> Public IPv4 Address

Reply:
For reference, https://aws.amazon.com/blogs/aws/new-aws-public-ipv4-address-charge-public-ip-insights/

Reply:
Refactored